src/cljs/org/broadinstitute/firecloud_ui/common/table_utils.cljs

```diff
@@ -45,56 +45,61 @@
+              [:div {:style (merge {:padding "2.15em 0em" :verticalAlign "middle"}
+                              layout-style)}
+               [:b {} (str left-num " - " right-num)]
+               (str " of " (pluralize num-rows-visible " result")
+                 (when-not (= num-rows-visible (:num-total-rows props))
+                   (str " (filtered from " (:num-total-rows props) " total)")))]
+              page-component
+              (style/create-unselectable
+                :div {:style (merge {:padding "1.6em 0em" :verticalAlign "middle" :textAlign "center"}
+                               layout-style)}
+                [:div {:style {:display "inline-block" :padding "0.55em 0.9em"
+                               :color (if allow-prev
+                                        (:button-blue style/colors)
+                                        (:border-gray style/colors))
+                               :cursor (when allow-prev "pointer")}
+                       :onClick (when allow-prev #(swap! state update-in [:current-page] dec))}
+                 (icons/font-icon {:style {:fontSize "70%"}} :angle-left)
+                 [:span {:style {:paddingLeft "1em"}} "Prev"]]
+                [:span {}
+                 (map (fn [n]
+                        (let [selected? (= n current-page)]
+                          [:div {:style {:paddingTop 5 :display "inline-block" :width 29 :height 24
+                                         :backgroundColor (when selected? (:button-blue style/colors))
+                                         :color (if selected? "white" (:button-blue style/colors))
+                                         :borderRadius (when selected? "100%")
+                                         :cursor (when-not selected? "pointer")}
+                                 :onClick (when-not selected? #(swap! state assoc :current-page n))}
+                           n]))
+                   (create-page-range current-page num-pages))]
+                [:div {:style {:display "inline-block" :padding "0.55em 0.9em"
+                               :color (if allow-next
+                                        (:button-blue style/colors)
+                                        (:border-gray style/colors))
+                               :cursor (when allow-next "pointer")}
+                       :onClick (when allow-next #(swap! state update-in [:current-page] inc))}
+                 [:span {:style {:paddingRight "1em"}} "Next"]
+                 (icons/font-icon {:style {:fontSize "70%"}} :angle-right)])
+              rows-component
+              [:div {:style (merge {:padding "2.15em 0em" :textAlign "right"}
+                              layout-style)}
+               "Display"
+               (style/create-select
+                 {:style {:width 60 :margin "0em 1em"}
+                  :onChange #(swap! state assoc
+                              :rows-per-page (-> % .-target .-value js/parseInt)
```

Comment:
This looks like a merge conflict bug. Take a look at this line in this view: https://github.com/broadinstitute/firecloud-ui/pull/259/files?w=1